protoc_plugin/README.md

```diff
@@ -106,21 +106,25 @@ re-export the relevant protocol buffer classes.
 Say we have the file `m1.proto` with the following content
 
 ```proto
+syntax = "proto3";
+
 message M1 {
-  optional string a;
+  optional string a = 1;
```

Comment:
I think in proto3 the default is to not put `optional`, the field will still not be required.

Reply:
I wasn't sure whether to delete or keep it. Should I remove it?

Reply:
I think so